drivers/volume/portworx/portworx_test.go

```diff
@@ -0,0 +1,111 @@
+package portworx
+
+import (
+	"fmt"
+	"testing"
+
+	apapi "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/pborman/uuid"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/pkg/aututils"
+	"github.com/portworx/torpedo/pkg/units"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCalculateAutopilotObjectSize(t *testing.T) {
+	inst := &portworx{}
+
+	type testCase struct {
+		rule           apapi.AutopilotRule
+		pool           node.StoragePool
+		node           node.Node
+		expectedSize   uint64
+		expectedToFail bool
+	}
+
+	testCases := []testCase{
```

Comment:
no volume UT yet?

Reply:
didn't make any logic changes to the volume function.